sdk/eventhub/Azure.Messaging.EventHubs/src/Metadata/EventHubProperties.cs

```diff
@@ -9,7 +9,7 @@ namespace Azure.Messaging.EventHubs.Metadata
     ///   A set of information for an Event Hub.
     /// </summary>
     ///
-    public sealed class EventHubProperties
+    public class EventHubProperties
```

Comment:
Somewhat orthogonal, I opened this up to allow library consumers to be able to mock when testing.